cmd/rook/ceph/osd.go

```diff
@@ -281,21 +281,17 @@ func commonOSDInit(cmd *cobra.Command) {
 }
 
 // use zone/region/hostname labels in the crushmap
-func getLocation(clientset kubernetes.Interface) (string, error) {
+func getLocation(clientset kubernetes.Interface) (string, string, error) {
```

Comment:
Named return values might be helpful here for understanding the function's use.

Reply:
The named labels change the dynamic of how variables are defined, so I don't see changing it. To me, named labels don't help with readability, but I could see that as a style preference.